spec/spec_helper_acceptance.rb

```diff
@@ -0,0 +1,36 @@
+require 'beaker-rspec'
+require 'beaker-puppet'
+require 'beaker/puppet_install_helper'
+require 'beaker/module_install_helper'
+
+run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
+install_module
+install_module_dependencies
+
+RSpec.configure do |c|
+
+  # Configure all nodes in nodeset
+  c.before :suite do
+    hosts.each do |host|
+      case host[:platform]
+      when /debian-7-amd64|debian-8-amd64|ubuntu-16.04-amd64/
+        on host, puppet('module', 'install', 'puppetlabs-apt')
+        pp = <<-EOS
+        package { 'netcat-openbsd' :
```

Comment:
you can install packages like this:
```ruby
install_package(host, 'netcat-openbsd') if fact('os.release.major')
```

Reply:
`netcat` was used in a previous version of tests. But now it is not. I'll remove this.